xray/formats/table.go

```diff
@@ -17,6 +17,18 @@ type VulnerabilityTableRow struct {
 	IssueId                   string                       `col-name:"Issue ID" extended:"true"`
 }
 
+type VulnerabilityScanTableRow struct {
+	Severity               string                   `col-name:"Severity"`
+	SeverityNumValue       int                      // For sorting
```

Comment:
The following applies to other places in the code as well - 
Description comments should be placed above the code they describe.

Reply:
Done!